forge-gui/res/tokenscripts/role_cursed.txt

```diff
@@ -0,0 +1,8 @@
+Name:Cursed Role
```

Comment:
The names of them are very most likely `Cursed Token`

Reply:
I think just `cursed `? 
![image](https://github.com/Card-Forge/forge/assets/67333662/8d54d761-4ab5-48fb-8817-e427f0443e71)

Reply:
yeah, but each Token needs to have "Token" at the end i think?

but yeah you might do it with just "Cursed" for now